packages/interface/src/features/projects/components/ProjectDetails.tsx

```diff
@@ -31,51 +32,53 @@ const ProjectDetails = ({ pollId, project, action = undefined }: IProjectDetails
-      </div>
-
-      <div className="mb-8 flex items-end gap-4">
-        <ProjectAvatar className="-mt-20 ml-8" rounded="full" size="lg" url={profileImageUrl} />
-      </div>
-
-      <div className="flex flex-col items-center justify-between px-2 sm:flex-row">
-        <Heading as="h3" size="3xl">
-          {metadata.data?.name}
-        </Heading>
-
-        {roundState === ERoundState.VOTING && (
-          <VotingWidget pollId={pollId} projectId={project.id} projectIndex={Number.parseInt(project.index, 10)} />
-        )}
-      </div>
-
-      <ProjectContacts author={payoutAddress} github={github} twitter={twitter} website={websiteUrl} />
-
-      <Markdown className="px-2 text-gray-400">{bio}</Markdown>
-
-      <div className="my-8 flex flex-col gap-8 px-2">
-        <p className="text-xl uppercase">
-          <b>Impact statements</b>
-        </p>
-
-        <ProjectDescriptionSection
-          contributions={metadata.data?.contributionLinks}
-          description={metadata.data?.contributionDescription}
-          title="contributions"
-        />
-
-        <ProjectDescriptionSection
-          description={metadata.data?.impactDescription}
-          fundings={fundingSources}
-          title="past grants and funding"
-        />
-
-        {action}
+    <div className="markdown-support relative flex flex-col gap-[30px]">
+      <Navigation pollId={pollId} projectName={metadata.data?.name ?? "project name"} />
+
+      <div className="flex flex-col gap-5">
+        <div className="flex flex-col">
+          <div className="overflow-hidden rounded-3xl">
+            <ProjectBanner url={bannerImageUrl} />
+          </div>
+
+          <div className="flex items-end gap-4">
+            <ProjectAvatar className="-mt-[75px] ml-8 !size-[105px]" rounded="full" size="lg" url={profileImageUrl} />
+          </div>
+        </div>
+
+        <div className="flex flex-col gap-[30px]">
+          <div className="flex flex-col items-center justify-between px-2 sm:flex-row">
+            <Heading as="h3" size="3xl">
+              {metadata.data?.name}
+            </Heading>
+
+            {roundState === ERoundState.VOTING && (
+              <VotingWidget pollId={pollId} projectId={project.id} projectIndex={Number.parseInt(project.index, 10)} />
+            )}
+          </div>
+
+          <ProjectContacts author={payoutAddress} github={github} twitter={twitter} website={websiteUrl} />
+
+          <Markdown components={markdownComponents}>{bio}</Markdown>
+        </div>
+
+        <div className="flex flex-col gap-5">
+          <h3 className="font-sans text-lg font-bold uppercase leading-[27px] dark:text-white">Impact statements</h3>
+
+          <ProjectDescriptionSection
+            contributions={metadata.data?.contributionLinks}
+            description={metadata.data?.contributionDescription}
+            title="contributions"
+          />
+
+          <ProjectDescriptionSection
```

Comment:
So this part is kind of weird, the impact description is not related to the funding description, maybe just remove the `impactDescription` here, and move it up to L67? (right after `Impact Statements`)

ps. it's also possible that there's only `impactDescription` but no `funding sources`, under this condition if the impact description is put in this section is really weird.